definitions/ext-dora/definition.yml

```diff
@@ -0,0 +1,41 @@
+domain: EXT
+type: DORA
+synthesis:
+  rules:
+    - identifier: doraName
+      name: doraName
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: DoraMetricsSample
+  tags:
+    doraName:
+      multiValue: false
+    repoName:
+      multiValue: false
+    deploymentFrequency:
+      multiValue: false
+    leadTimeForChanges:
+      multiValue: false
+    timeToRestoreService:
+      multiValue: false
+    changeFailureRate:
+      multiValue: false
+    projectId:
+      multiValue: false
+    repoUrl:
+      multiValue: false
+    team:
+      multiValue: true
```

Comment:
nit: true is the default so feel free to remove these lines

Reply:
removed and updated 👍🏾